splunk-otel-collector.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: splunk-otel-collector
+  version: 0.113.0
+  epoch: 0
+  description: Splunk OpenTelemetry Collector is a distribution of the OpenTelemetry Collector. It provides a unified way to receive, process, and export metric, trace, and log data for Splunk Observability Cloud
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/signalfx/splunk-otel-collector
+      tag: v${{package.version}}
+      expected-commit: afd65d06be2164c95fb50a9d6ab2f42af9e86383
+
+  - uses: go/build
+    with:
+      modroot: cmd/otelcol
+      packages: .
+      ldflags: -X github.com/signalfx/splunk-otel-collector/internal/version.Version=v${{package.version}}
```

Comment:
it looks like there are two ldflags should be defined: https://github.com/signalfx/splunk-otel-collector/blob/main/Makefile#L30-L32
this one `-X go.opentelemetry.io/collector/internal/version.Version=${{package.version}}` needs to be added as well.

Reply:
Both ldflags added, thanks for catching this